browser/playlist/playlist_service_factory.cc

```diff
@@ -81,7 +81,8 @@ PlaylistServiceFactory::~PlaylistServiceFactory() = default;
 KeyedService* PlaylistServiceFactory::BuildServiceInstanceFor(
     content::BrowserContext* context) const {
   DCHECK(media_detector_component_manager_);
-  return new PlaylistService(context, media_detector_component_manager_.get());
+  return new PlaylistService(context, media_detector_component_manager_.get(),
+                             nullptr);
```

Comment:
why does this take a third parameter now when we are setting it as null?

Reply:
Have a follow up patch. Will update.